README.md

```diff
@@ -96,15 +96,33 @@ If you don't have a Discord bot token, you'll have to create one. To create a Di
    ![PNG - Bot Created Successfully](../media/created-bot.png?raw=true)
 7) Now pressing the "Copy" button will copy your `Token` to your clipboard.
 
-...and that's it!
+...and that is all!
+
+## Restricting Bot Access to Channels
+
+For testing purposes, you may wish to restrict the bot's access to specific channels. 
+
+When you add the bot to your Discord server, the system will automatically create a role for that bot. In order to restrict access to channels, it is important that the bot and bot role do **not** have the "Administrator" permission (this overrides channel-specific permissions). 
+
+For each channel you want keep the bot *out* of, you need to do the following:
+
+1. Select the gear icon next to the channel.
+2. Select "Permissions". 
+3. Next to the Roles/Members, click the `+` icon. 
+4. Add the bot's role.
+5. The added role should now be selected.
+6. To the right, go through the permissions list and set each permission to the red "X".
+   The most important permissions to turn off are "Read message history", "Read messages", and "Send messages".
+   
```

Comment:
Add another step for the that mentions clicking "save changes"

Reply:
Done and done. 
Though the Discord app won't let you exit that screen without either choosing "Save Changes" or "reset". It shakes and gets all mad at you 😛

Reply:
Best if we instruct people to save without making discord angry :P